tests/v2/unit/adapters/torch/lightning/anomalib/helpers/dummy_dataset.py

```diff
@@ -100,29 +99,28 @@ def __init__(self, task_type: TaskType) -> None:
             ],
         )
         self.config = OmegaConf.create({"dataset": {"image_size": [32, 32]}})
+        
+        self.item_ids: list = [item.id for item in self.dataset]
```

Comment:
Would it be possible to annotate the list? Like `list[<type>]`?

Reply:
I clarified at 0e4de99acf5e9b1e54d3375cba358c4c02d964b2.